src/styles/_card.scss

```diff
@@ -0,0 +1,132 @@
+@use 'variables';
+
+.card {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: variables.$mainColor;
```

Comment:
Ensure that `variables.$mainColor` is defined in the `variables` file. If it's not defined, this will cause an error during compilation.